src/swift/SwiftAPIView.xcodeproj/xcshareddata/xcschemes/SwiftAPIView.xcscheme

```diff
@@ -52,17 +52,21 @@
       </BuildableProductRunnable>
       <CommandLineArguments>
          <CommandLineArgument
-            argument = "--source=/Users/jairmyree/Desktop/azure-sdk-for-ios/sdk/communication/AzureCommunicationChat/Source"
+            argument = "--source=/Users/travisprescott/repos/azure-sdk-for-ios/sdk/communication/AzureCommunicationChat/Source"
```

Comment:
Is this correct to have your local directory path here?

Reply:
Yes. Anyone who clones it would just change it for their own machine, or ignore it entirely.